versioned_plugins.rb

```diff
@@ -413,16 +410,15 @@ def lazy_create_output_folder(output_asciidoc)
   # param plugin_names_by_type: map of lists {:input => [beats, tcp, ...]}
   # return list of triples (type, alias, target) es: ("input", "agent", "beats")
   def load_alias_definitions_for_target_plugins(plugin_names_by_type)
-    alias_url = URI('https://raw.githubusercontent.com/elastic/logstash/master/logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml')
-    alias_yml = Net::HTTP.get(alias_url)
-    yaml = YAML::safe_load(alias_yml) || {}
+    alias_plugin_util = Util::AliasPluginUtil.new
+    yaml = alias_plugin_util.fetch_alias_mappings
```

Comment:
Nitpick (repeat): `yaml` local variable does not contain _YAML_.

Reply:
Provided descriptive names for all relative logics.